vp/workflow/models.py

```diff
@@ -94,8 +114,9 @@ def read_json(self):
             OSError: on file error
             NetworkXError: on issue with loading JSON graph data
         """
-
-        with open(self.file_path, 'r') as file:
+        print(fs.location)
+        print(self.file_path)
```

Comment:
Maybe better to use logger instead of print? In case those print statements are necessary, if not we can maybe jusr remove them.